### PR TITLE
Small code improvements

### PR DESCRIPTION
**Pull request by t4im**
These changes don't change behavior, just improve speed a little and clean up a bit.
With the exception of the last commit, that simply makes sure, the right electrode is returned when dug.

Is it intentional, that the cooled ore blocks get destroyed as well inside in the furnace?

I really like this mod idea. Simple concept with great emergent gameplay potential.

### CODE REVIEW

File: init.lua [hunk 1]
```diff
@@ -644,80 +586,41 @@ minetest.register_abm({
...
+			return
 		end
-	
+
 		-- don't cool near active electrodes
 		if nil ~= minetest.find_node_near(pos, 4, {mn..":electrode_on"}) then
 			return
 		end
 
-		-- don't cool near heater bricks
```
Comment (yzziizzy) [inline init.lua:661]: Was this removed on purpose? Or did I miss it somewhere else in the diff?

**Comment by yzziizzy** [comment]
Thanks for the code improvements :) Sorry for the slow response, it's vacation season.

File: init.lua [hunk 2]
```diff
@@ -46,74 +46,42 @@ minetest.register_craftitem(mn..":refractory_clay_brick", {
...
 
 minetest.register_node(mn..":refractory_brick", {
-    description = "Refractory Brick",
-    tiles = { "default_brick.png^[colorize:white:120" },
-    is_ground_content = true,
-    groups = {cracky=1, level=3, refractory=3},
-    sounds = default.node_sound_stone_defaults(),
-}) 
-
-minetest.register_node(mn..":furnace_heater", {
```
Comment (yzziizzy) [inline init.lua:76]: furnace heater removed? Or is this a weird diff artifact from merging from an older version?

**Comment by yzziizzy** [comment]
There was no intention for cooled ore blocks to be destroyed when in range of an active electrode ("inside the furnace" for all intents). The destruction ABM was just slapped together in a few minutes to force the use of refractory materials for crucibles. Ideally they would melt into molten ore sources.

File: init.lua (hunk 1 above)
Comment (t4im) [reply init.lua:661]: Oh, my bad. That must have slipped out during rebasing against the last commit.
Adding it back in.

**Comment by yzziizzy** [comment]
Cool, thanks.

**Pull request merged**